src/main/java/com/xiaomi/infra/pegasus/client/PegasusClient.java

```diff
@@ -88,6 +89,7 @@ public PegasusClient(Properties config) throws PException {
     this.cluster = Cluster.createCluster(config);
     this.tableMap = new ConcurrentHashMap<String, PegasusTable>();
     this.tableMapLock = new Object();
+    this.metaList = cluster.getMetaList();
```

Comment:
Why do you add a variable of metaList? you can use cluster.getMetaList() directly.

Reply:
The variable is used for [PegasusTable](https://github.com/Shuo-Jia/pegasus-java-client/blob/detail-log/src/main/java/com/xiaomi/infra/pegasus/client/PegasusTable.java#L39-L43)